src/usability-testing/usability-testing.tex

```diff
@@ -43,90 +41,82 @@ \section{Participants}\label{sec:participants}
-direct solution.
-Peter mentioned that the feedback should provide future moves along with the current move.
+The main focus point with the usability testing was the learning aspect of the project, because that's the goal of the
+project.
+The feedback on the learning aspect was not very positive.
 Ronja didn't immediately understand what the Stockfish feedback was, but after using Stockfish a little, she lost
 interest in it.
-This indicates that the current Stockfish implementation has a lot of room for improvement on what feedback it provides
-and how to provide it to the player.
+Clara thought that the Stockfish feedback was too direct, and Peter mentioned that the feedback should provide future
+moves along with the current move.
+None of them thought that the feedback was helpful.
+This indicates that the feedback from Stockfish has a lot of room for improvement, as only providing the best move is
+not enough for learning, because players wouldn't understand why it's the best move.
+The team was already aware of this issue and had discussed a potential solution, which was conveyed to the users in the
+next focus point.
 
-Clara also mentioned that the game should have more teaching tools, such as puzzles and guides to how to play chess.
-The team considered these features given the learning nature of the game, but there are already existing solutions that
-provide them, so they were not prioritized, instead focusing on the coaching aspect of the game.
+\subsection{Potential solution}\label{subsec:potential-solution}
 
-Lastly, the users were asked about the idea of using machine learning along with Stockfish to provide feedback.
+As a potential solution, users were asked about the idea of using a large language model along with Stockfish to provide
+feedback.
 The rationale behind this question was due to a prior discussion within the team, which will be later discussed in
 Section~\ref{sec:large-language-model-implementation}.
-Clara and Ronja both liked the idea of using machine learning, while Peter was more skeptical.
+Clara and Ronja both liked the idea of using a large language model, while Peter was more skeptical.
 He did not particularly like the idea due to the lack of control over the large language model's answers, so the team
 cannot be guaranteed that it'll provide correct information.
 Clara however thinks that in combination with Stockfish it could provide accurate information whilst also giving more
 natural responses and explanations.
+She and Ronja think that a natural response would help a lot with understanding the feedback.
+An example given was that the player could ask follow-up questions, which the large language model could answer,
+providing a more interactive learning experience.
 
-% textidote: ignore begin % due to section being too short
+\subsection{Other feedback}\label{subsec:other-feedback}
```

Comment:
I still think this subsection is a little irrelevant as it does not have much to do with learning. To make it more relevant you could make a little intoduction like:
"The usertests also uncovered other issues that had hindered the learning for the test subjects" - or something like that.

Reply:
It's true that it has little to do with learning, which is why we don't focus on this subsection as much as the previous two. I still wanted to keep it here because I didn't want to disregard any feedback we might have gotten.

Reply:
Agreed. I just think there should be a learning angle to it

Reply:
But this other feedback has little to do with learning

Reply:
I agree that it does not seem to have that much to do with learning. 

Right now IMO though, the subsection is a little over half a page that describes what is essentially useless information in regards to solving the problem statement.

However, why it is relevant can still be implied through an introduction like the one i suggested, which would solve my relevancy concern.

Of course it may just be me who thinks so. If others do not agree we can just go with what is written :)

Reply:
I've added an introduction and considerably shortened the section itself. I mostly moved their final thoughts to the conclusion, which seems to fit better. 

I'm really against the idea of saying that this feedback relates to learning experience. That's just incorrect for a big portion of it. Meanwhile I'm also against removing it outright, because as I said, I don't want to disregard any feedback we've gotten. Reports must include everything related to the project, with in-scope issues being the main focus. So I think it's fine as it is now.

Reply:
Okay, it should be good now